scripts-dev/build_debian_packages

```diff
@@ -179,11 +179,20 @@ if __name__ == "__main__":
         action="store_true",
         help="skip running tests after building",
     )
+    parser.add_argument(
+        "--show-dists-json",
+        action="store_true",
+        help="instead of building the packages, just list the dists to build for, as a json array",
+    )
     parser.add_argument(
         "dist",
         nargs="*",
         default=DISTS,
         help="a list of distributions to build for. Default: %(default)s",
     )
     args = parser.parse_args()
-    run_builds(dists=args.dist, jobs=args.jobs, skip_tests=args.no_check)
+    if args.show_dists_json:
+        json.dump(DISTS, sys.stdout)
+        print()
```

Comment:
I find `print(json.dumps(DISTS))` easier to read but ymmv

Reply:
fair